mmu-to-hats/catalog_functions/desi_transformer.py

```diff
@@ -111,7 +112,7 @@ def dataset_to_table(self, data):
 
         # 4. Add boolean features
         for f in self.BOOL_FEATURES:
-            columns[f] = pa.array(data[f][:].astype(bool))
+            columns[f] = pc.invert(pa.array(data[f][:].astype(bool)))
```

Comment:
I think it would be better to invert that single feature explicitly, so the code makes that intention more explicit.